cassandra-utils/src/main/java/org.ballerinalang.cassandra/CassandraDataSource.java

```diff
@@ -583,10 +605,7 @@ private ConsistencyLevel retrieveConsistencyLevel(String consistencyLevel) {
     private enum SocketOptionsParam {
         // int params
         CONNECT_TIMEOUT_MILLIS("connectTimeoutMillis"), READ_TIMEOUT_MILLIS("readTimeoutMillis"), SO_LINGER(
-                "soLinger"), RECEIVE_BUFFER_SIZE("receiveBufferSize"), SEND_BUFFER_SIZE("sendBufferSize"),
-
-        // boolean params
-        KEEP_ALIVE("keepAlive"), REUSE_ADDRESS("reuseAddress"), TCP_NO_DELAY("tcpNoDelay");
```

Comment:
Can't we use these enum constants anywhere?

Reply:
AFAIK they are not used now.